calblueprint/dream-rails
# Casey/sessions

**What's in this PR?**
- Adds a Session model to keep track of sessions for a course. (i.e. each session has a day, start, and end time -- ordered by session 'number' when displayed on course page)
- Association between Sessions and Courses (session belongs to course, course has many sessions)
- CRUD for sessions in session controller
- Update Course controller to add sessions to a course, update, and display sessions. 
- Add dream_id for students

**To test**
- Run 'rake db:migrate'
- Mainly just see that nothing is broken on course creation.
- Hard to test because all interactive visual things on frontend

**Coming in next PR**
Dynamically add course sessions on course form
